package.json

```diff
@@ -48,9 +49,9 @@
   "devDependencies": {
     "@babel/node": "^7.0.0",
     "babel-eslint": "^10.0.1",
+    "babel-plugin-add-module-exports": "^1.0.0",
```

Comment:
Is `babel-plugin-add-module-exports` being used anywhere? If no, please remove it and update the `lock` file. If yes, what are we using it for?